app/views/tags/edits.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="tag-show-container">
+    <%= render partial: 'header', locals: { tag: @tag, active_tab: :edits } %>
+</div>
+
+<div id="tag-homepage-edits-table-container" class="top-1em">
```

Comment:
interesting. you could have included this as a partial on the `show` view and `#show` action. curious how you evaluated the decision to put it here instead of there. (makes sense that the other one is switching off of `tagable_categories` where as this is fundamentally different. but the fact that this goes on the `show` page and is accessed on a route `/tags/:id/edits` (similar in form to `tags/:id/:tagable_category` might argue for thinking of it as part of the `#show` action/view?) i don't have hugely strong opinions on the matter, just curious what informed your thinking on it.

Reply:
that's basically it.

1) the edits page isn't a tagable_category and I didn't want to break the sensible logic and reasoning of what the #show route does

2) I decided that table better suited the edits page, instead of a list, meaning there wasn't that much code to share. And what could be shared  -- the header -- i turned into a partial.